src/govuk/components/file-upload/_index.scss

```diff
@@ -11,6 +11,16 @@
     padding-top: $component-padding;
     padding-bottom: $component-padding;
 
+    // The default file upload button in Safari does not
+    // support setting a custom font-size. Set `-webkit-appearance`
+    // to `button` to drop out of the native appearance so the
+    // font-size is set to 19px
+    // https://bugs.webkit.org/show_bug.cgi?id=224746
+    &::-webkit-file-upload-button {
```

Comment:
Looking at [what normalize.css does](https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L303-L311) for file uploads, it looks like they set `font: inherit` as well, which might be worth doing here too?